pkg/network/tracer/offsetguess/tracer.go

```diff
@@ -752,8 +773,37 @@ func (t *tracerOffsetGuesser) Guess(cfg *config.Config) ([]manager.ConstantEdito
 	if err == nil && State(t.status.State) == StateReady {
 		return t.getConstantEditors(), nil
 	}
-
-	eventGenerator, err := newTracerEventGenerator(t.guessUDPv6)
+	var curNS netns.NsHandle
```

Comment:
Could we move the ns setup into its own helper function that returns the current ns? Current ns could be the new or the root ns depending on if we create a new net ns.